### ISSUE TITLE
[BUG] Bottles freeze the entire system on Ubuntu 18.04 (hwe 5.4.0-97-generic)

### ISSUE THREAD

**Issue by venetin** [bottlesdevs/Bottles#1015]
Please don't fill issue for a Windows executable that doesn't work in Bottles. Our software is a Wine prefix manager. If a Windows application doesn't work, it could be a Wine problem (**Wine is NOT a software developed by Bottles**) or a misconfiguration of the wineprefix (take a look at WineHQ: https://www.winehq.org) or ProtonDB: https://www.protondb.com for more information on the software you are trying to run). 

We will support the installation of Windows software, only through our official installers: https://github.com/bottlesdevs/Bottles/issues/55 (when these are ready).
Please DO NOT open issues for this reason. Issues should be used to report Bugs and request New Features for Bottles.

**Describe the bug**
When I try to run an exe in Bottles it starts to use 100% CPU and the system freezes. There is no way to run e.g. ctrl+alt+F1. The only way to "unfreeze" the system is to use the power button. This leads to broken file system. Then I need to run fsck from the Ubuntu recovery (initramfs). I managed to start the the exe once after several fsck but on the second run it freeze again. The very same exe files run OK under wine and POL. This laptop is my work laptop and there not much software installed. On my other laptop with tons of software (same bottles config) and ubuntu 20.04 there are no such issues. I tested both laptops with the default configs for games and for software. The results are the same. Did't try with custom configs.

**Installation**
 - Method: Flatpak
 - Version 2022.1.28-trento-3

**To Reproduce**
1. Just run any .exe file
2. Wait 5 seconds and the system will freeze (the exe is not started, at least there is no GUI)

**Expected behavior**
I expect the exe file to start.
**Screenshots**
Not possible to take screenshot

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04 with HWE (kernel 5.4.0-97-generic)
 - Version 18:04
 - Display server: Xorg
**Additional context**

**Comment by mirkobrombin** [comment]
Is virtual desktop enabled?

**Comment by venetin** [comment]
No, the virtual desktop is not enabled.I tested with default bottles settings for game and for software.Venetin Konstsntinov11:15, 3 февруари 2022 г., Mirko Brombin ***@***.***>:
Is virtual desktop enabled?

**Comment by venetin** [comment]
So it seems that the issue is present also on Ubuntu 20.04 with Xorg. The kernel is the same 5.4.0-97-generic which is actually the hwa kernel of 18.04. I made some tests on 20.04 and it crashed on the first exe which I tried to run. So it seems that Bottles can't be used with this configuration.

**Comment by mirkobrombin** [comment]
I created a VM with Ubuntu 20.04 but haven't tested yet

**Comment by venetin** [comment]
I think that the issue is probably hardware related (or kernel/drivers related) and a bare metal is needed for testing. But since it brakes the filesystem I am not a volunteer for testing right now. The other possibility is that it is something with the flatpak. This is my first and only installed flatpak app. But on 18.04 the flatpak is from a ppa and on 20.04 it is the one from the Ubuntu repos....

**Comment by mirkobrombin** [comment]
Can also be an hardware problem. btw I prefer to try on myself before

**Comment by luni3359** [comment]
I'm also experiencing a full system freeze after running an application, not even REISUB works. 

@venetin How does it break the filesystem though? Everything works for me as normal after hard-resetting.

**Comment by venetin** [comment]
@luni3359 On 20.04 with i3 6006U CPU I did not expirience filesystem issues but I tried only once. I don't want to brake something. On 18.04 on i7 CPU the filesystem start to break on the second or third test. Maybe the time when you press and hold the power button is essential. If you wait a little bit until it finishes to do something the filesystem will be OK. But on the second or third test I started to press the power button as soon as the system became unresponsive. The last test was on 20.04 and I wait around 20 sconds before I hit the power button. Please note that on 20.04 with i3 CPU the fan is starting to rotate very rare. But on my i7 laptop during the freeze the fan was at 100%. That's why i decided not to wait and pressed the button immidetely from the second or third test on....

**Comment by luni3359** [comment]
You're right, perhaps it's best if you didn't try again.

**Comment by test2a** [comment]
i am on 2022.1.28-trento-4
and i tried to change "runner" to a bottle in preferences and that just hangs the system entirely.

same for downloading and installing a runner in global preferences.

i have only been experiencing this debilitating crashes for the last few weeks and i cannot find much to log it but this is consistently happening to me in various parts of the app

Operating System: KDE neon 5.24
KDE Plasma Version: 5.24.0
KDE Frameworks Version: 5.90.0
Qt Version: 5.15.3
Kernel Version: 5.13.0-28-generic (64-bit)
Graphics Platform: X11
Processors: 4 × Intel® Core™ i7-4600U CPU @ 2.10GHz
Memory: 7.7 GiB of RAM
Graphics Processor: Mesa DRI Intel® HD Graphics 4400

**Comment by mirkobrombin** [comment]
After several tests I can confirm that I am unable to reproduce this problem in any way. Can you give me the precise steps or make a video somehow? (Even from the phone if it crashes ..)

**Comment by luni3359** [comment]
I felt adventurous today and decided to run the windows program again since Bottle has gone through a few more updates. 

I clicked the run button but this time around the system didn't freeze, but the application did. Resource usage didn't seem to change at all either. However I noticed that clicking other buttons, like the plus button for adding a new application, didn't do anything. Bottles would just freeze for about 5 minutes and then become responsive again, without doing anything as if I hadn't clicked anything.

I tried testing this with the current data in my .var folder, but I tried with a fresh start and I couldn't add any new applications to test this.  @nSneerfulBike are you using everything in English, with no non-English characters? That's the only thing I assume might be the issue because the non-latin characters in the terminal here are sideways.

![The Japanese characters are sideways on the terminal](https://user-images.githubusercontent.com/16832398/157513112-ddee6a73-158b-405b-b1f5-13c521a8d43d.png)

**Comment by nSneerfulBike** [comment]
> I felt adventurous today and decided to run the windows program again since Bottle has gone through a few more updates.
> 
> I clicked the run button but this time around the system didn't freeze, but the application did. Resource usage didn't seem to change at all either. However I noticed that clicking other buttons, like the plus button for adding a new application, didn't do anything. Bottles would just freeze for about 5 minutes and then become responsive again, without doing anything as if I hadn't clicked anything.
> 
> I tried testing this with the current data in my .var folder, but I tried with a fresh start and I couldn't add any new applications to test this. @nSneerfulBike are you using everything in English, with no non-English characters? That's the only thing I assume might be the issue because the non-latin characters in the terminal here are sideways.
> 
> ![The Japanese characters are sideways on the terminal](https://user-images.githubusercontent.com/16832398/157513112-ddee6a73-158b-405b-b1f5-13c521a8d43d.png)

When I tried, I did both in a VM (only English for both system language and keyboard layout) and on my PC (English for system language, IT and EN-US for keyboard layout) and the behavior was exactly the same. I installed Bottles through Flatpak since the .deb package didn't work back then. What method have you used?

**Comment by luni3359** [comment]
Wait so the vm caused your entire system to crash? That sounds pretty big lol. 

Hmm... I don't think it can be a locale problem anymore. I'm running Bottles through Flatpak. A few comments above I put more details about my setup.

**Comment by nSneerfulBike** [comment]
> Wait so the vm caused your entire system to crash? That sounds pretty big lol.

Luckily not, it just crashed the VM. It crashed the system when I installed Bottles on my system tho, so same behavior.
I'll try again when I'll have some time and let you know with a couple more tests

**Comment by mirkobrombin** [comment]
@francescomasala can you try this too in a vm?

**Comment by nSneerfulBike** [comment]
While doing some testings, the situation has worsened a lot now. Rufus started correctly, but WhatsApp and Discord just couldn't be started, and when I wanted to make a few more testings I saw a Bottles update on Flatpak and installed it. Well, now the app just gets stuck on the blank home page without loading anything and any button apart from the close button won't do anything. I even tried to reinstall everything from scratch, even the VM, but Bottles won't go past the blank home screen now

Ubuntu 21.10 on latest apt/snap updates. Both X11 and Wayland don't work

**Comment by mirkobrombin** [comment]
Latest release doesn’t hit the wine backend

**Comment by mirkobrombin** [comment]
We are not able to reproduce this in anyway. I'm closing for now, feel free to add more comments.

**Comment by martin-desktops** [comment]
I have the same issue while playing The Sims 4 through Origin, flatpak. No matter weather its Wayland or X11, or if the game is windowed or fullscreen. It occurs after an irregular period - sometimes 20 minutes, sometimes it’s an hour.  I would love to provide more information, though a video recording can be problematic because as I said it takes long time waiting for the freeze. When the freeze occurs the only thing I am able to do is to manually shutdown the computer with physical button.
